src/bauhaus/bauhaus.c

```diff
@@ -1648,15 +1653,15 @@ static void dt_bauhaus_draw_quad(dt_bauhaus_widget_t *w, cairo_t *cr)
     switch(w->type)
     {
       case DT_BAUHAUS_COMBOBOX:
-        cairo_translate(cr, width - darktable.bauhaus->quad_width * .5f, height * .4f);
```

Comment:
I had put that setting to have a smaller arrow, and nearly same size as classic combo (check on collapsible section of import module). The arrow is now a little too big.
But I've tested by putting back 0.4 and indeed, now it's not good as it break vertical align you succeed to implement.

It should be possible anyway to reduce a little the arrow. Could you update that?

Reply:
Not that bad to me, it makes the collapsible section a bit more visible.

Reply:
Alic answer to me in Matrix and to change size correctly, that was not there. So, this change is good so.